skimage/measure/_regionprops.py

```diff
@@ -622,6 +625,21 @@ def regionprops_table(label_image, intensity_image=None,
     """
     regions = regionprops(label_image, intensity_image=intensity_image,
                           cache=cache)
+
+    if len(regions) == 0:
+        label_image = np.zeros(tuple([3] * len(label_image.shape)), dtype=int)
+        label_image[tuple([1] * len(label_image.shape))] = 1
+        if intensity_image is not None:
+            intensity_image = label_image.copy()
```

Comment:
The intensity image needs to have the same type as the input intensity image, so I suggest:

```suggestion
            intensity_image = np.zeros(label_image.shape, dtype=intensity_image.dtype)
```